R/frmt_utils.R

```diff
@@ -67,6 +67,24 @@ print.frmt_combine <- function(x,...){
   cat(format(x),sep = "\n")
 }
 
+
+#' @export
+format.frmt_when <- function(x, ...){
+  frmt_str <- cat(
+    "< frmt_when | ", "\n",
+    "LHS: ", paste0(map(x, rlang::f_lhs) %>% map(format) %>% unlist(), collapse = ", "), "\n",
```

Comment:
```suggestion
    "LHS: ", paste0(map(x, f_lhs) %>% map(format) %>% unlist(), collapse = ", "), "\n",
```
Can you remove the double colons. If this breaks the code it means you need to import these functions you can so that by adding 
```r
#' @importFrom rlang f_lhs rf_rhs 
```